source/scripts/tests/BatTest.js

```diff
@@ -30,5 +31,13 @@ export default function BatTest() {
         // being spawned in.
     })
 
-    // Put assertions here that test your code!
+    // Had to use JSON.stringify to compare objects
```

Comment:
Hmm.. We should find a better way of comparing two positions, so we don't have to stringify them again next time.